pkg/cloud/network_test.go

```diff
@@ -276,4 +276,41 @@ var _ = Describe("Network", func() {
 			Ω(client.ResolveNetwork(dummies.CSCluster, &dummies.ISONet1)).Should(Succeed())
 		})
 	})
+
+	Context("Network Semi-Integ Tests", func() {
```

Comment:
New integ test that could have prevented the whole fiasco. So, at least we shouldn't fail this way again.